### PR TITLE
Added option to not wipe bed info.  Also reformatted to follow yapf config.

### PR DESCRIPTION
**Pull request by dulacarnold**

### CODE REVIEW

File: scripts/wipe_sensitive_info_db.py [hunk 1]
```diff
@@ -10,64 +10,64 @@
...
-    for alteration in ALTERATIONS.items():
-        affectations = ",".join([f"{k} = {v}" for (k, v) in alteration[1]])
-        query = f"UPDATE {alteration[0]} SET {affectations};"
-        print(query)
-        conn.execute(query)
-    conn.commit()
-    conn.close()
+  conn = sqlite3.connect(path)
+  cur = conn.cursor()
+  if FLAGS.keep_beds:
```
Comment (rprimet) [inline scripts/wipe_sensitive_info_db.py:47]: Is there a `not` missing here?

File: scripts/wipe_sensitive_info_db.py [hunk 2]
```diff
@@ -10,64 +10,64 @@
...
+    ALTERATIONS["bed_counts"].extend([("n_covid_free", 2), ("n_covid_occ", 4)])
+  for alteration in ALTERATIONS.items():
+    affectations = ",".join([f"{k} = {v}" for (k, v) in alteration[1]])
+    query = f"UPDATE {alteration[0]} SET {affectations};"
+    print(query)
```
Comment (rprimet) [inline scripts/wipe_sensitive_info_db.py:52]: The print statement should also probably be in a `if not FLAGS.quiet:` guard but maybe that's a separate PR?

File: scripts/wipe_sensitive_info_db.py (hunk 1 above)
Comment (dulacarnold) [reply scripts/wipe_sensitive_info_db.py:47]: completely, good catch

File: scripts/wipe_sensitive_info_db.py (hunk 2 above)
Comment (dulacarnold) [reply scripts/wipe_sensitive_info_db.py]: Done.

**Pull request merged**